lib/utils/forkedblockchain.js

```diff
@@ -327,6 +303,28 @@ ForkedBlockchain.prototype.getCode = function(address, number, callback) {
   });
 };
 
+ForkedBlockchain.prototype.getLookupAccount = function(trie) {
```

Comment:
Reason for change: we needed to be able to lookup an account based on a trie, not a block number.